api/src/test/java/org/openmrs/module/ohricore/dao/OHRICoreDaoTest.java

```diff
@@ -3,21 +3,26 @@
  * v. 2.0. If a copy of the MPL was not distributed with this file, You can
  * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
  * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
```

Comment:
@smallgod Delete this test it is not useful or needed anywhere, also remove all references to the Item class

Reply:
@ssmusoke done!